cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -822,6 +822,9 @@ def _validate_embedding_dict(self):
 
         obsm_with_x_prefix = 0
         for key, value in self.adata.obsm.items():
+            if " " in key:
```

Comment:
i think any keys that have whitespace in them should be invalid, regardless of whether it's an `X_{suffix}` key